docs/set-up.asciidoc

```diff
@@ -6,34 +6,90 @@ you must first {apm-server-ref-v}/configuration-rum.html[enable the RUM endpoint
 
 Once the APM Server endpoint has been configured, you can:
 
-* <<install-the-agent>>
+* <<usage>>
 * <<configuring-cors>>
 
-[[install-the-agent]]
-=== Install the Agent
+[[usage]]
+=== Usage
 
-There are three ways to use the APM RUM Agent:
+There are three ways to include the APM RUM Agent:
+
+APM RUM Agent can be added to your web page in one of two ways
```

Comment:
"There are **three** ways..."
_followed by_
"...in one of **two** ways"

Let's get rid of the number to ease maintenance. How about: "There are multiple was to add the APM RUM Agent to your web page:"

Reply:
Forgot to remove the first line, Yeah your suggestion seems to be better.